### PR TITLE
Fix for mishandling of VCF INFO fields of type flag

### PR DESCRIPTION
**Pull request by alexramos**
Addresses #291

### CODE REVIEW

File: oncotator/output/RecordBuilder.py [hunk 1]
```diff
@@ -289,6 +289,11 @@ def addInfo(self, sampleName, ID, num=None, dataType="String", val=None, isSplit
             self._determineVal2FixedNumField(self._info, ID, nalts, isSplit, val)
         elif num == 0:  # num is either true or false
             if ID not in self._info:
+
```
Comment (LeeTL1220) [inline oncotator/output/RecordBuilder.py]: Are you sure you mean ``val == ''``?  And not ``val is None``?  Also, were you able to make a unit test?  That should be really easy.

Comment (alexramos) [inline oncotator/output/RecordBuilder.py]: Yes I'm sure.  The new 1000g ds was already preventing a unitest from passing so it didn't seem necessary to write a new one.  From now on, we should rerun all tests using the new db-dir before releasing a new corpus...

Comment (alexramos) [inline oncotator/output/RecordBuilder.py]: The comment is confusing I'll change it.

**Comment by LeeTL1220** [comment]
You want to merge into release-1.5, not develop.  This is a hotfix.

**Pull request closed without merging**